components/base/Button.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+import {
+  WHITE_PRIMARY,
+  PURPLE_PRIMARY_BUTTON,
+  PURPLE_PRIMARY_HOVER,
+  PURPLE_PRIMARY_ACTIVE,
+  PINK_SECONDARY_BUTTON,
+  PINK_SECONDARY_HOVER,
+  PINK_SECONDARY_ACTIVE,
+  GREY_4,
+  GREY_9,
+  GREY_5,
+  PURPLE_DARK,
+} from "../colors";
+import { BOX_SHADOW_6 } from "../shadowDepths";
+
+type Props = {
+  text: string;
+  size: string;
```

Comment:
For `size` and `variant` we could use [union types](https://www.typescriptlang.org/docs/handbook/advanced-types.html#union-types) to specify the accepted values. For example:

`size: "tiny" | "small" | "standard" | "large";`

Reply:
Thanks for catching that! 👍

Reply:
This [commit](https://github.com/Dwolla/dev-portal/pull/15/commits/7d47ff01d96b5204451ad2cd7f5ec6d82aa5b3c7) should fix that.